### PR TITLE
#1 commit React+Tailwind setup. 2 PR requests

### PR DESCRIPTION
**Pull request by NastiaMospan**
Deleted  App.css, index.css and 
         App.jsx and followed instruction to set up tailwind.

### CODE REVIEW

**Comment by RobyOneJ** [comment]
Just an heads up, do you guys remember Alexander's saying that they actually made a mistake in the notes and should not have removed app.jsx (and of course its app.css) but actually use it to handle a few things like: Banner, NavBar, Outlet and Footer? 

I did follow his suggestion for my frontend and made it easier. Just think about it - but I am ok with your approach too.

File: .gitignore
Comment (Sandz-Studio) [inline .gitignore:1]: .env needs to be in git ignore

File: src/App.jsx
Comment (Sandz-Studio) [inline src/App.jsx:1]: we need to keep app.jsx

**Review by Sandz-Studio — COMMENTED** [review]
missing index.html as well. we keep app.jsx as we use it to return <NavBar /> <Outlet /> and <FooterBar />

File: .gitignore
Comment (RobyOneJ) [reply .gitignore:1]: Have you checked the gitignore from the copied repo, cuz backend code already had it excluded.
I went to check immediately when cloned.

Index is required.

Comment (Sandz-Studio) [reply .gitignore:1]: We need it in front as well, that I where the api url does. Needs to be there for both.

Comment (Sandz-Studio) [reply .gitignore:1]: Yep Index is in my merge request

**Pull request closed without merging**